pkg/gooq/expression_test.go

```diff
@@ -130,6 +130,26 @@ var expressionTestCases = []TestCase{
 		ExpectedStmt: "table1.uuid_column NOT IN ($1, $2)",
 		Arguments:    []interface{}{uuid.Nil, uuid.Nil},
 	},
+	{
+		Constructed:  Table1.DecimalColumn.Add(Int64(42)),
+		ExpectedStmt: "table1.decimal_column + $1",
+	},
+	{
+		Constructed:  Table1.DecimalColumn.Sub(Int64(-42)),
+		ExpectedStmt: "table1.decimal_column - $1",
+	},
+	{
+		Constructed:  Table1.DecimalColumn.Mult(Int64(42)),
+		ExpectedStmt: "table1.decimal_column * $1",
+	},
+	{
+		Constructed:  Table1.DecimalColumn.Div(Int64(0)),
+		ExpectedStmt: "table1.decimal_column / $1",
+	},
+	{
+		Constructed:  Table1.DecimalColumn.Sqrt(),
+		ExpectedStmt: "|/ table1.decimal_column",
+	},
```

Comment:
can we move these tests closer to the other `Table1.DecimalColumn` tests

Reply:
On it :+1: